### ISSUE TITLE
 Upload multiple files accessibility audit 

### ISSUE THREAD

**Issue by kellyChex** [samvera/hyrax#3981]
### Descriptive summary

QA Testing for Hyrax
Hyrax Version: 3.0.0.pre.rc1
Hyrax Branch: master
Hyrax Revision: 48ed925
Nurax Version
ENV: production
SHA: 40b47b0d71af1b958e016ebe49212c1740683133
Branch: 2019-09-12-10-00
Updated: 12 Sep 2019 15:00:37

In testing for the Hyrax 3.0 release, the following points are actions that were discovered during a FireFox tab test and axe accessibility audit. Note, only critical issues are being reported at this time.

**id attribute value must be unique**
![Screen Shot 2019-09-12 at 4 56 08 PM](https://user-images.githubusercontent.com/9621070/64828645-3dbe7b80-d57e-11e9-9305-978e1a49ce2f.png)

**ARIA role must be appropriate for the element**
![Screen Shot 2019-09-12 at 4 56 53 PM](https://user-images.githubusercontent.com/9621070/64828681-57f85980-d57e-11e9-857f-89247970f810.png)

**Heading levels should only increase by one**
![Screen Shot 2019-09-12 at 4 57 51 PM](https://user-images.githubusercontent.com/9621070/64828708-7bbb9f80-d57e-11e9-9bd5-812d0bcf9e27.png)

**Aside must not be contained in another landmark**
![Screen Shot 2019-09-12 at 4 58 30 PM](https://user-images.githubusercontent.com/9621070/64828733-94c45080-d57e-11e9-9afb-682786f7db33.png)

**Main landmark must not be contained in another landmark**
![Screen Shot 2019-09-12 at 4 59 19 PM](https://user-images.githubusercontent.com/9621070/64828770-b1f91f00-d57e-11e9-9c14-c61b8adac2ef.png)

**Document must have one main landmark**
![Screen Shot 2019-09-12 at 4 59 55 PM](https://user-images.githubusercontent.com/9621070/64828792-c6d5b280-d57e-11e9-83b2-6fffd8fc12cc.png)

**Ensures landmarks are unique**
![Screen Shot 2019-09-12 at 5 00 32 PM](https://user-images.githubusercontent.com/9621070/64828816-db19af80-d57e-11e9-8a62-0c336d975a4d.png)

![Screen Shot 2019-09-12 at 5 01 47 PM](https://user-images.githubusercontent.com/9621070/64828887-07cdc700-d57f-11e9-8782-00d4000d22aa.png)

![Screen Shot 2019-09-12 at 5 02 08 PM](https://user-images.githubusercontent.com/9621070/64828901-14521f80-d57f-11e9-923f-8e27f4c81681.png)

**Radio inputs with the same name attribute value must be part of a group**

![Screen Shot 2019-09-12 at 5 02 49 PM](https://user-images.githubusercontent.com/9621070/64828932-2b910d00-d57f-11e9-81c6-7fb717eeac72.png)

### Steps to reproduce the axe accessibility audit

1. Install the Axe plugin or addon from Deque (https://addons.mozilla.org/en-US/firefox/addon/axe-devtools/ is the firefox version).
2. Open https://nurax-dev.curationexperts.com/concern/images/tq57nr202/edit?locale=en#files and click Add folder and select multiple files 
3. Open the inspector (right-click, and select "Inspect element")
4. Click on the 'Axe' tab
5. Click 'Analyze'

### Tab Test:

In Firefox and Safari, unable to tab Feature, Analytics, Edit, Delete, etc. 

Chrome working fine. 

1. 
https://cl.ly/43fba47d0336/download/Screen%252520Recording%2525202019-09-12%252520at%25252005.03%252520PM.mov

Link: https://nurax-dev.curationexperts.com/concern/generic_works/th83kz52k/edit?locale=en#files 

Unable to tab through Descriptions, Files, Relationships, Sharing, and left hand navigation 

2. 
https://cl.ly/6c19a1dfe079/download/Screen%252520Recording%2525202019-09-12%252520at%25252004.42%252520PM.mov

**Comment by rjkati** [comment]
Tab test passes in Hyrax 3.5.0.

axe plugin is unavailable for Firefox, but reported issues are not present in Hyrax 3.5.0 using WAVE for Firefox.
